DigitalLearningSolutions.Web/ViewModels/TrackingSystem/CentreManagerDetailsViewModel.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem
+{
+    using System.ComponentModel.DataAnnotations;
+    using DigitalLearningSolutions.Data.Models;
+
+    public class CentreManagerDetailsViewModel
+    {
+        public CentreManagerDetailsViewModel(){ }
+
+        public CentreManagerDetailsViewModel(Centre centre)
+        {
+            FirstName = centre.ContactForename;
+            LastName = centre.ContactSurname;
+            Email = centre.ContactEmail;
+            Telephone = centre.ContactTelephone;
+        }
+
+        [Required(ErrorMessage = "Please enter a first name.")]

```

Comment:
It's not major as I think there will be another ticket to go though and check all of these, but I think we are going with the format 'Enter a first name' rather than 'please'